plugins/obs-filters/obs-filters.c

```diff
@@ -72,12 +77,17 @@ bool obs_module_load(void)
 	obs_register_source(&expander_filter);
 	obs_register_source(&luma_key_filter);
 	obs_register_source(&luma_key_filter_v2);
+#if LIBNVVFX_ENABLED
+	if (load_nvvfx())
+		obs_register_source(&nvidia_greenscreen_filter_info);
+#endif
 	return true;
 }
 
 #ifdef LIBNVAFX_ENABLED
 void obs_module_unload(void)
 {
 	unload_nvafx();
+	unload_nvvfx();
```

Comment:
I think this should be guarded by LIBNVVFX_ENABLED

Reply:
oh right, thx; fixed in last force push.